tests/unit/modules/test_zypperpkg.py

```diff
@@ -94,7 +94,7 @@ def test_list_upgrades(self):
         }
         with patch.dict(
             zypper.__salt__, {"cmd.run_all": MagicMock(return_value=ref_out)}
-        ):
+        ), patch.object(zypper.__zypper__, "_is_rpm_lock", return_value=False):
```

Comment:
I guess this should be like:
```suggestion
        ), patch.object(zypper.__zypper__, "_is_rpm_lock", MagicMock(return_value=False)):
```

as `return_value` is a parameter for `MagicMock`, right?

Reply:
Good to know, thanks for clarifying! :+1: